contracts/rewards/src/state.rs

```diff
@@ -178,6 +178,39 @@ pub struct Epoch {
     pub block_height_started: u64,
 }
 
+impl Epoch {
+    /// Returns the current epoch. The current epoch is computed dynamically based on the current
+    /// block height and the epoch duration. If the epoch duration is updated, we store the epoch
+    /// in which the update occurs as the last checkpoint
+    pub fn current(
+        stored_params: &StoredParams,
```

Comment:
Could we rename `StoredParams` to `LatestParams`? I find that name very confusing and doesn't really tell you anything about it's content

Reply:
renamed to `ParamsSnapshot` and `last_updated` to `created_at` after discussion